src/components/OrgSettings/General/OrgUpdate/ImagePicker.tsx

```
@@ -0,0 +1,62 @@
+}) => {
+  const [previewImage, setPreviewImage] = useState<string | null>(
+    defaultImage || null,
+  );
+
+  /**
+   * Handles image file selection, converts it to Base64, and updates the preview.
+   */
+  const handleImageChange = async (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ): Promise<void> => {
+    const file = e.target.files && e.target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)